orchestra/google_apps/service.py

```diff
@@ -129,3 +129,47 @@ def delete_folder(self, folder_id):
         except errors.HttpError as error:
             logger.exception('An error occured: %s', error)
             return None
+
+    def list_folder(self, folder_id):
```

Comment:
we test the service by way of the convenience methods, and by creating a mocked out google service:

https://github.com/unlimitedlabs/orchestra/blob/master/orchestra/tests/helpers/google_apps.py#L64

by faking the google service's responses, he was able to test all of the `service.py` code that calls it